index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+
+  <!-- link to fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li class="nav-item logo">Bennyjoez</li>
+        <li class="nav-item logo">
+          <i class="fa-solid fa-bars"></i>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <span class="greeting">
+        Hey There. <br>
+        I'm Benson
+      </span>
```

Comment:
- [ ] [REQUIRED] Good job so far!  Please, implement the main title (e.g., "I'm Marianne ...") `h1` tag not a `span` 🙏🏼 If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.